alphadia/peakgroup/config_df.py

```diff
@@ -124,7 +124,9 @@ def __init__(
         self.feature_weight = feature_weight
 
 
-class HybridCandidateConfig(JITConfig):
+class HybridCandidateConfig(
+    JITConfig
+):  # TODO rename to HybridCandidateScoringHyperparameters
```

Comment:
I think we can simplify the name and drop the `Hybrid` to `CandidateScoring` and